packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,140 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+---
+
+## Design
+
+Simple accordion group
+
+```
+<nimble-accordion>
+    <nimble-accordion-item>
+        <span slot="heading">Panel one</span>
+        Panel one content
+    </nimble-accordion-item>
+    <nimble-accordion-item expanded>
+        <span slot="heading">Panel two</span>
+        Panel two content
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <span slot="heading">Panel three</span>
+        Panel three content
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+Complex accordion group with components in slots and single expand-mode
+
+```
+<nimble-accordion expand-mode="single">
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion one</div>
+        <nimble-select slot="end">
+            <nimble-option value="1">Option 1</nimble-option>
+            <nimble-option value="2">Option 2</nimble-option>
+            <nimble-option value="3">Option 3</nimble-option>
+        </nimble-select>
+            Accordion one content
```

Comment:
I might just remove the various "Accordion <x> content", as I don't think it really helps illustrate the API/functionality of the accordion. It's just string literal content mixed with other component content, but its presence is more confusing than anything (I get that this is how the FAST example illustrates the API, but they don't provide other content like we are here).